package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "plaid",
-  "version": "3.1.1",
+  "version": "4.0.0",
```

Comment:
How about we make this a minor bump to 3.2.2? There don't seem to be any breaking changes, so I don't see a need to major bump it

Reply:
Interesting -- I don't think I updated this myself. Does it update as part of `make setup`? regardless, going to 3.2.0

Reply:
:zap